yogstation/code/modules/reagents/reagent_containers/gummies.dm

```diff
@@ -0,0 +1,191 @@
+	color = "#863333"
+
+/obj/item/reagent_containers/gummy/drugs
+	list_reagents = list(/datum/reagent/drug/space_drugs = 10, /datum/reagent/consumable/orangejuice = 3, /datum/reagent/consumable/sugar = 2)
+	color = "#E78108"
+
+/obj/item/reagent_containers/gummy/meth
+	desc =  "A sweet chewable gummy bear! Could this one be lemon flavor?"
+	list_reagents = list(/datum/reagent/drug/methamphetamine = 2, /datum/reagent/consumable/banana = 13)
+	color = "#ffdf4e"
+
+/obj/item/reagent_containers/gummy/nitro
+	list_reagents = list(/datum/reagent/nitrous_oxide = 10, /datum/reagent/consumable/applejuice = 3, /datum/reagent/consumable/sugar = 2)
+	color = "#20a000"
+
+/obj/item/reagent_containers/gummy/mime
+	desc =  "A sweet chewable gummy bear! Smells of sweetness and...nothing?"
+	list_reagents = list(/datum/reagent/consumable/nothing = 13, /datum/reagent/consumable/sugar = 2)
+	color = "#313131"
+
+/obj/item/reagent_containers/gummy/vitamin
+	desc = "You shouldn't see this! tell a coder if you do!"
+	list_reagents = list(/datum/reagent/consumable/nutriment/vitamin = 15)
+	color = null
+
+/obj/item/reagent_containers/gummy/vitamin/berry
+	desc = "A sweet chewable gummy bear!"
+	list_reagents = list(/datum/reagent/consumable/nutriment/vitamin = 4, /datum/reagent/consumable/berryjuice = 6, /datum/reagent/consumable/sugar = 5)
+	color = "#770b0b"
+
+/obj/item/reagent_containers/gummy/vitamin/orange
+	desc = "A sweet chewable gummy bear!"
+	list_reagents = list(/datum/reagent/consumable/nutriment/vitamin = 4, /datum/reagent/consumable/orangejuice = 6, /datum/reagent/consumable/sugar = 5)
+	color = "#E78108"
+
+/obj/item/reagent_containers/gummy/vitamin/lime
+	desc = "A sweet chewable gummy bear!"
+	list_reagents = list(/datum/reagent/consumable/nutriment/vitamin = 4, /datum/reagent/consumable/limejuice = 6, /datum/reagent/consumable/sugar = 5)
+	color = "#22ff00"
+
+/obj/item/reagent_containers/gummy/vitamin/lemon
+	desc = "A sweet chewable gummy bear! Is it lemon, or banana?..."
+	list_reagents = list(/datum/reagent/consumable/nutriment/vitamin = 4, /datum/reagent/consumable/lemonjuice = 6, /datum/reagent/consumable/sugar = 5)
+	color = "#ECFF56"
+
+/obj/item/reagent_containers/gummy/vitamin/banana
+	desc = "A sweet chewable gummy bear! Is it lemon, or banana?..."
+	list_reagents = list(/datum/reagent/consumable/nutriment/vitamin = 4, /datum/reagent/consumable/banana = 11)
+	color = "#fdff98"
+
+/obj/item/reagent_containers/gummy/vitamin/apple
+	desc = "A sweet chewable gummy bear!"
+	list_reagents = list(/datum/reagent/consumable/nutriment/vitamin = 4, /datum/reagent/consumable/applejuice = 6, /datum/reagent/consumable/sugar = 5)
+	color = "#20a000"
+
+/obj/item/reagent_containers/gummy/vitamin/honey
+	desc = "A sweet chewable gummy bear! This one smells strongly of Honey!"
+	list_reagents = list(/datum/reagent/consumable/nutriment/vitamin = 4, /datum/reagent/consumable/honey = 11)
```

Comment:
Honey has the tendency to break eown into sugar, and jt does it quite often, so this may be a one way trip to obesity as is anything else with a "normal" amount of honey in it.

Reply:
hmmm....maybe you have a point...not sure what to use as filler though, I dont want people injecting them with stuff, and I cant increase the vitamins or we run into Donk Pocket issues